suite-common/wallet-core/src/accounts/accountsReducer.ts

```diff
@@ -78,6 +78,11 @@ export const prepareAccountsReducer = createReducerWithExtraDeps(
             .addCase(accountsActions.updateAccount, (state, action) => {
                 update(state, action.payload);
             })
+            .addCase(accountsActions.renameAccount, (state, action) => {
+                const { accountKey, accountLabel } = action.payload;
+                const renamedAccount = state.find(account => account.key === accountKey);
```

Comment:
nit: it's not renamed account 
```suggestion
                const accountByAccountKey = state.find(account => account.key === accountKey);
```

Reply:
fixed in 0168e3f56